app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,18 @@
 <resources>
-    <string name="app_name">Code Challenge</string>
+    <string name="app_name">Movie Finder</string>
+    <string name="details">Details</string>
     <string name="movie_and_genres">%s - %s</string>
+    <string name="poster_description">Poster of the movie</string>
+    <string name="backdrop_description">Backdrop image of the movie</string>
+    <string name="title">Title</string>
+    <string name="release">Release</string>
+    <string name="genre">Genre</string>
+    <string name="overview">Overview</string>
+    <string name="search">🔎 Search</string>
+    <string name="favorite_description">Button to favorite movie</string>
+    <string name="favorites">Favorites</string>
+
+    <string-array name="warning">
```

Comment:
Why did you use a `<string-array>` here?

Reply:
Using a `<string-array>` you can call multiple values by the same string name. As it was clear that the project is intended to increase in size, it's good to keep in mind that other warnings will probably be necessary.